### DISCUSSION TITLE
Boilerplate for Newcomer!?

### DISCUSSION THREAD

**Discussion by AERONAUT-AUT** [fullstackhero/blazor-wasm-boilerplate#232] [General]
Hi, 
I started with the BlazorHero Template and was more or less successful with this template. 
For me as a newcomer it was way easier to understand as the FSH Boilerplate.
Furthermore it looks like the coding effort is also a lot more on the fsh template as on the BlazorHero one.

For me it would be interesting to know, what was your basis of decision making to swap to the fsh boilerplate?

BRGDS
Stefan

**Comment by kirkey** [comment]
I am afraid this template documentation was abandoned months ago. There are too many inquiries but nobody's interested to support. Better find another template like ABP etc.
